lib/Service/RecipeService.php

```diff
@@ -137,13 +137,10 @@ public function checkRecipe($json) {
 
         // Make sure that "keywords" is an array of unique strings
         if(isset($json['keywords']) && is_string($json['keywords'])) {
-          $keywords = trim($json['keywords']);
-          $keywords = trim($keywords, ',');
-          $keywords = trim($keywords)
+          $keywords = trim($json['keywords'], " \0\t\n\x0B\r,");
```

Comment:
I think, this is the same as 
```suggestion
          $keywords = trim($json['keywords']);
```
[trim documentation](https://www.php.net/manual/en/function.trim.php)

Reply:
No.

If you notice, we're adding a comma to the list of characters to be trimmed; otherwise yes, it would be the same.